website/source/docs/providers/aws/r/s3_bucket.html.markdown

```diff
@@ -51,8 +51,9 @@ The following arguments are supported:
 
 The website object supports the following:
 
-* `index_document` - (Required) Amazon S3 returns this index document when requests are made to the root domain or any of the subfolders.
+* `index_document` - (Required, unless using `redirect_all_requests_to`) Amazon S3 returns this index document when requests are made to the root domain or any of the subfolders.
```

Comment:
I wasn't sure of the best way to word this. We could just say this is optional, and put a note at the end of the line saying it's required if not specifying `redirect_all_requests_to`.